src/HTTPSock.cpp

```diff
@@ -181,6 +188,12 @@ void CHTTPSock::ReadLine(const CString& sData) {
             // X-Forwarded-For list in both cases use it as the endpoind
             m_sForwardedIP = sIP;
         }
+    } else if (sName.Equals("X-Forwarded-CertFP:")) {
```

Comment:
I can't find anything about this name.

Reply:
Yeah, it's not standardized anywhere. X-Forwarded-For is standardized by convention but as far as I know there isn't anything similar to TLS fingerprints, as this sort of thing isn't as widely done. This would have to go in documentation somewhere.